src/main/kotlin/no/nav/familie/ks/sak/barnehagelister/BarnehagebarnService.kt

```diff
@@ -74,14 +53,34 @@ class BarnehagebarnService(
 
     private fun getCorrectSortBy(sortBy: String): String =
         when (sortBy.lowercase()) {
-            "endrettidspunkt" -> "endret_tid"
+            "endrettidspunkt" -> "endretTid"
```

Comment:
Denne må hete det samme som kolonnenavnet for at group by skal fungere som forventet. Altså at dersom nøyaktig samme barnehagebarn blir sendt inn to ganger så viser vi bare fram den med nyligst endret_tid